lib/va_healthcare_messaging/api/triage_teams.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'common/models/collection'
+
+module VaHealthcareMessaging
```

Comment:
Do we want to maybe just call this SM for secure_messaging, instead va_healthcare_messaging? previously we needed it that way because it was a gem or library, but now that it's all in this repository it seems less important, and the shorter the better maybe?